broader-topics/ecotax.md

```diff
@@ -0,0 +1,65 @@
+# SPECIFICATIONS - ECOTAX
+
+## WHAT IS THE ECOTAX
+
+The ecotax is a contribution in France to finance the collect, the recycling and the depollution of a piece of furniture or used electrical and electronic equipment. 
+The ecotax is not a tax per se, but a fixed mandatory contribution (the actual term is "eco-participation") that must be paid by the customer.
+Then it’s donated by the seller to a collection agency. 
+
+The amount of the ecotax is fixed and does not depend on the product price.
+The ecotax is subject to VAT.
+
+## WHERE IS THE ECOTAX IN PRESTASHOP
+
+The ecotax can be toggled in the "Tax Options" section from the _International > Taxes_ page.
+Once enabled, a new drop-down is displayed containing all the tax rules, allowing the merchant to choose the VAT rate that will be applied on the ecotax.
```

Comment:
Will this line be deleted once the PR for [#22827](https://github.com/PrestaShop/PrestaShop/issues/22827) is done ?

Reply:
See https://github.com/PrestaShop/prestashop-specs/pull/188/commits/62ddf4a7ac3125ac199a71240607fa499f155950